### PR TITLE
feat(minio): import updated minio package and add tag on file upload

### PR DESCRIPTION
**Pull request by joremysh**
Because

- run logging metadata retention

This commit

- import updated minio package and add tag on file upload

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -248,3 +239,14 @@ func ParseConfigFlag() string {
 
 	return *configPath
 }
+
+const (
+	FreePlanExpiryTag = "free-plan-expiry"
```
Comment (jvallesm) [inline config/config.go:244]: ❓ Can we use another tag (e.g. `default-expiry`) or is the logic tied to this constant? I think it'd be better if we could  avoid using a cloud-only term like "plan"

**Review by jvallesm — COMMENTED** [review]
Looks good, just a small comment about terminology

**Comment by jvallesm** [comment]
🙏 Can you remove the commit list from the PR description. It'll end up in the merge commit message. Thanks!

> 
    chore(minio): replace minio package with common package in x
    feat(minio): import updated minio package and check subscription plan
    feat(minio): import updated minio package and add tag on file upload

File: config/config.go (hunk 1 above)
Comment (joremysh) [reply config/config.go]: updated and removed messages in PR description

**Review by jvallesm — APPROVED** [review]

**Pull request merged**